dool

```diff
@@ -2242,10 +2246,13 @@ def cprint(var, ctype = 'f', width = 4, scale = 1000):
 
     ### Add unit to output
     if unit:
-        if c != -1 and round(var) != 0:
-            ret += cunit + units[c]
-        else:
-            ret += char['space']
+        try:
+            if c != -1 and round(var) != 0:
+                ret += cunit + units[c]
+            else:
+                ret += char['space']
+        except OverflowError:
+            pass
```

Comment:
Instead of just `pass` it should also add a space to it, otherwise this line breaks the columns.
```suggestion
            ret += char['space']
```

Reply:
done.

Reply:
You didn't push anything.

Reply:
Yeah, didn't  notice the force required after the rebase on master.  Updated  now.